lib/Tapable.js

```diff
@@ -53,33 +53,37 @@ Tapable.mixin = function mixinTapable(pt) {
 	copyProperties(Tapable.prototype, pt);
 };
 
-Tapable.prototype.applyPlugins = function applyPlugins(name) {
+Tapable.prototype.applyPlugins = function applyPlugins(name, param1, param2) {
 	if(!this._plugins[name]) return;
```

Comment:
Jep, just checked Chrome 61 (which is what Node 8 will eventually be based on in the end), and same picture there:

![Chrome 61 results](https://user-images.githubusercontent.com/815988/29247945-a7df322c-800b-11e7-8834-2e02d93df2ba.png)